storage2/src/lib.rs

```diff
@@ -1,4 +1,5 @@
-use jmt::WriteOverlay;
+#![recursion_limit = "256"] // required for TCT
+
```

Comment:
We should remove this, and the dependency of the `storage2` crate on the `tct` crate.